daisy_workflows/image_import/windows/run_startup_scripts.cmd

```diff
@@ -16,12 +16,28 @@ REM limitations under the License.
 REM Give the network time to initialize.
 ping 127.0.0.1 -n 60
 
-echo "Translate: Starting image translate..." > COM1:
+set image_import_reg_path="HKLM\Software\ImageImport"
 
-REM Restart the system in 5 minutes. Translate.ps1 will cancel this restart when it starts.
+for /f "tokens=3" %%a in ('reg query %image_import_reg_path% /v StartupScriptRunsCount') do set startup_script_runs_count=%%a
+if %ERRORLEVEL% EQU 0 (
+  reg delete %image_import_reg_path% /v StartupScriptRunsCount /f
+) else (
+  set startup_script_runs_count=0
+)
+set /a startup_script_runs_count=%startup_script_runs_count% + 1
+reg add %image_import_reg_path% /v StartupScriptRunsCount /t REG_DWORD /d %startup_script_runs_count%
```

Comment:
I don't recall seeing registry changes in other parts for image import; I don't have enough Windows development experience to know whether or not this is an idiomatic approach.

I noticed that import creates temporary files, however, and cleans them up after import. So if you wanted to stay consistent with what already exists, then you could use a temporary file. Here's an example of where a temporary file gets removed:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/91fe21ea4c88821b199fdef40602e3fa65967bf6/daisy_workflows/image_import/windows/translate.ps1#L393